src/styles/style-helpers.spec.tsx

```diff
@@ -13,14 +13,81 @@
  * limitations under the License.
  */
 
-import { light } from '@sumup/design-tokens';
-
-import * as StyleHelpers from './style-helpers';
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import { light, Theme } from '@sumup/design-tokens';
+
+import { create } from '../util/test-utils';
+
+import {
+  cx,
+  shadowSingle,
+  shadowDouble,
+  shadowTriple,
+  headingKilo,
+  headingMega,
+  headingGiga,
+  headingTera,
+  headingExa,
+  headingPeta,
+  headingZetta,
+  subHeadingKilo,
+  subHeadingMega,
+  textKilo,
+  textMega,
+  textGiga,
+  disableVisually,
+  hideVisually,
+  focusOutline,
+  clearfix,
+  hideScrollbar,
+  inputOutline,
+} from './style-helpers';
 
 describe('Style helpers', () => {
+  const red = (theme: Theme) => css`
+    color: ${theme.colors.r500};
+  `;
+  const green = (theme: Theme) => css`
+    color: ${theme.colors.g500};
+  `;
+
+  describe('cx', () => {
+    it('should call each style function with the theme', () => {
+      const actual = create(<div css={cx(red, green)} />);
+
+      expect(actual).toMatchInlineSnapshot(`
+        .circuit-0 {
+          color: #D23F47;
+          color: #8CC13F;
+        }
```

Comment:
I wonder if we should have a test with helpers that make sense to combine, like a color and a font size maybe? Just to make the example closer to reality

Reply:
They'd work the same way. This test simply ensures that all styles are applied, no matter what they are. The documentation will have better examples.